_config.yml

```diff
@@ -1,7 +1,7 @@
 name: 18F Visual Identity
 
 repo:
-- name: 18F Brand
+- name: 18F Visual Identity
```

Comment:
I _think_ changing this will change what shows up on the GitHub tab name, but I'm not sure. From what I could tell, making this change at least didn't break anything in the repo :)